webapp/channels/src/components/admin_console/access_control/modals/job_details/searchable_sync_job_channel_list.tsx

```diff
@@ -239,7 +239,7 @@ const SearchableSyncJobChannelList = (props: Props) => {
     }
 
     const input = (
-        <div className='filter-row filter-row--full'>
+        <div className='filter-row'>
```

Comment:
This change wasn't strictly necessary for this, but it seems like these `filter-row` and `filter-row--full` ended up behaving exactly the same, so I got rid of the redundant one